app/models/event_instance.rb

```diff
@@ -20,6 +20,18 @@ class EventInstance < ActiveRecord::Base
 
   validate :dont_update_after_finished, on: :update
 
+  def self.active_hangouts
+    select(&:live?)
+  end
+
+  def self.extract_yt_id(yt_video_url)
```

Comment:
so this new method would ideally have a rspec unit test to check that it was working ...